configure/olsconfigure-api.adoc

```diff
@@ -0,0 +1,16 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="olsconfig-api"]
+= OLSConfig API reference
+include::_attributes/common-attributes.adoc[]
+:context: olsconfig-api
+
+toc::[]
+
+`OLSConfig` is the Schema for the {ols-long} configuration, which defines the underlying deployments.
```

Comment:
I would stick with _and_. Using _which_ here really means _the {ols-long} configuration, and defines the underlying deployments._  And, it keeps it consistent with the above. 
```suggestion
`OLSConfig` is the Schema for the {ols-long} configuration, and defines the underlying deployments.
```

Reply:
Thanks!

Done.